add Khant Bio

khantm02 opened the pull request:
- [x] clone the home repo
- [x] create a new branch "khant"
- [x] run npm install 
- [x] create a new file "khantm02.md" 
- [x] add my image to img folder
- [x]  add info about me to "khantm02.md" 
- [x] npm run format, npm run format:check
- [x] npm run spell-check
- [x] npm run lint:ls, npm run lint:md
- [x] add my changes to the staging area
- [x] commit my changes
- [x] push my code to GitHub
- [x] make pull request

denisrossi commented on student-bios/khantm02.md:
I guess you want to add some markdown content :stuck_out_tongue_closed_eyes: 

You can find [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) the basic syntax for the Github flavor of markdown

khantm02 replied:
Hi coach, I have written the README file. The problem is I cloned the repo again, created a new branch, committed the changes and tried to push it again. I cannot push it because it is showing me an error. I tried doing that way as I could not figure out how to access my branch for this pull request again. I hope you understood my point.

samirm00 commented:
Hello @khantm02 please create a deliverable issue and link it to your `PR`

bermarte commented:
The file is empty, write some markdown.

bermarte reviewed (changes requested):
All the images should go to the img folder (no logos, delete it), you can't use HTML in Markdown. Your profile image is too dark.

khantm02 commented:
Hi coach, I have fixed these issues. I just listed the technology stack instead of photos. I deleted the logo album as well. The reason I used html tag was to resize my profile image. I hope everything is good now.
